File: app/spring-webflux/src/main/java/com/jtlapp/jvmvsjs/springwebflux/WebApp.java
package com.jtlapp.jvmvsjs.springwebflux;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {
		"com.jtlapp.jvmvsjs.springwebflux", "com.jtlapp.jvmvsjs.webfluxlib"
})
public class WebApp {

	public static void main(String[] args) {
		SpringApplication.run(WebApp.class, args);
	}

}

File: app/spring-webflux-kotlin/src/main/kotlin/com/jtlapp/jvmvsjs/springwebfluxkotlin/WebApp.kt
package com.jtlapp.jvmvsjs.springwebfluxkotlin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class WebApp

fun main(args: Array<String>) {
	runApplication<WebApp>(*args)
}
